packages/cloud_firestore/lib/src/query.dart

```diff
@@ -53,7 +53,10 @@ class Query {
     StreamController<QuerySnapshot> controller; // ignore: close_sinks
     controller = StreamController<QuerySnapshot>.broadcast(
       onListen: () {
-        _handle = Firestore.channel.invokeMethod<int>(
+        // TODO(amirh): remove this on when the invokeMethod update makes it to stable Flutter.
+        // https://github.com/flutter/flutter/issues/26431
+        // ignore: strong_mode_implicit_dynamic_method
+        _handle = Firestore.channel.invokeMethod(
```

Comment:
There seems to be a merging/rebase issue where invokeMethod's type parameters got reverted back in this PR in multiple places.

Reply:
You're right, I've fixed this